docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1509,6 +1560,16 @@ must restart the server for that change to take effect.
 |`fhirServer/notifications/kafka/enabled`|Y|Y|
 |`fhirServer/notifications/kafka/topicName`|N|N|
 |`fhirServer/notifications/kafka/connectionProperties`|N|N|
+|`fhirServer/notifications/nats/enabled`|N|N|
+|`fhirServer/notifications/nats/cluster`|Y|N|
+|`fhirServer/notifications/nats/channel`|N|N|
+|`fhirServer/notifications/nats/clientId`|N|N|
+|`fhirServer/notifications/nats/servers`|Y|N|
+|`fhirServer/notifications/nats/useTLS`|N|N|
+|`fhirServer/notifications/nats/truststoreLocation`|Y|N|
+|`fhirServer/notifications/nats/truststorePassword`|Y|N|
+|`fhirServer/notifications/nats/keystoreLocation`|Y|N|
+|`fhirServer/notifications/nats/keystorePassword`|Y|N|
```

Comment:
have we tested that this is tenant specific? 
how is this valid given the fact the channel is not tenant?

Reply:
Actually the channel could be tenant-specific, since it is configurable.  Probably worth a discussion about how the notification system would behave in a multi-tenant environment.  If there is fhir-server-config.json and nats notification class instance per tenant, then channel and client-id could be tenant-specific.  I was viewing that column from the perspective of whether a single tenant would need to change that value to something specific for that tenant, which is where the 'N's came from.

Reply:
what we mean by "tenant-specific" here is whether the properties will be read from the tenant-specific config file if they are set there.  In this case, the nats notifier reads these properties during server startup (FHIRServletContextListener.contextInitialized), so we want an N in this column.  I've already committed the change, so this one is now resolved.